gulp-tasks/publish.js

```diff
@@ -24,7 +25,14 @@ async function publish_sign_in_check() {
   await execa('npm', ['whoami']);
 }
 
+function dist_tag_check() {
```

Comment:
This is also checked inside of `publish_lerna`, but failing early (before the `test` task runs) seemed ideal.